CHALLENGE 2.0/def,multiple_3_calc.py

```diff
@@ -0,0 +1,8 @@
+def mul_calc(num):
+    num=float(input("enter value here; "))
+    arrayA=range(3,100,5)
```

Comment:
Are you sure `range(3,100,5)` would solve any array of numbers? Because I don't why you are using fixed numbers here.

Reply:
yes , i have changed *arg  in the def function so it can collect  an any array of numbers